rcutils/logging.py

```diff
@@ -54,7 +54,8 @@
 skipfirst_doc_lines = [
     'The first log call is being ignored but all subsequent calls are being processed.']
 throttle_params = OrderedDict((
-    ('time_source_type', 'The time source type of the time to be used'),
+    ('time_source_type', 'Function that returns rcutils_ret_t and expects a '
```

Comment:
How about renaming this parameter to match the new semantic, e.g. `get_time_point_value`?

Reply:
That goes against https://github.com/ros2/rcutils/pull/183#discussion_r330247026. Do we want to be explicit about the parameter being a function or do we want to hide it for the sake of API ergonomics? I'm slightly inclined towards the latter but it's not a strong stand.

Reply:
The function parameter is being used as a function:

> if (time_source_type(&__rcutils_logging_now) != RCUTILS_RET_OK) {

There is no way you can just pass `RCUTILS_STEADY_TIME` (assuming it isn't a macro defining a function).

Reply:
That's precisely what I'm proposing, an alias for that function so that it reads better. But it's a detail and orthogonal to the actual macro parameter name.

Reply:
And that's what @BMarchi implemented in https://github.com/ros2/rcutils/pull/183/commits/e1b4344f90c79523908ece4454399e5743585eea. WDYT @dirk-thomas ?

Reply:
My previous comment still applies: it is not a time source so it should not be named `time_source` imo.